docs/developer-docs/latest/development/export-import.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Data export-import transfer system - Strapi Developer Docs
+description: Import, export, and transfer data using the Strapi CLI
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/export-import.html
+---
+
+# Data Export-Import Transfer System <BetaBadge />
+
+:::callout 🚧 Feature under development
+The data export-import transfer system in under development. Not all use cases are covered by the initial release. You can provide feedback about desired functionality on the [Strapi feedback website](feedback.strapi.io).
+:::
+
+Occasionally you need to move data out of or into a Strapi instance. The data export-import transfer system allows you to efficiently extract data from an existing instance or archive and import that data into a separate instance. Strapi provides CLI-based commands that allow you to export and import data using providers. Common use cases include:
+
+- creating a data backup,
+- restoring data from a backup
+<!-- transferring data between environments such as staging and production,
+ moving assets from one hosting solution to another, such as locally hosted to an S3 bucket. -->
+
+ The following documentation details examples of how to use the `strapi export` and `strapi import` commands.
+
+:::strapi Using the Command Line Interface (CLI)
+The `strapi export` CLI command and all of the available options are listed in the [Command Line Interface documentation](/developer-docs/latest/developer-resources/cli/CLI#strapi-export.md).
+:::
+
+## Export data using the CLI tool
+
+The `strapi export` command by default exports data as an encrypted and compressed `tar.gz` file. The default export command exports:
+
+- the project configuration,
+- entities,
+- links,
+- assets,
+- schemas,
+- `metadata.json` file.
```

Comment:
Should we mention what it is? Some additional information logged about the export